src/main/kotlin/com/codereview/iou/controller/IouController.kt

```diff
@@ -0,0 +1,43 @@
+package com.codereview.iou.controller
+
+import com.codereview.iou.model.dto.*
+import com.codereview.iou.model.entity.PromissoryNote
+import com.codereview.iou.service.PromissoryNoteService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/v1/iou")
+class IouController {
+
+    @Autowired
+    lateinit var service: PromissoryNoteService
+
+    @GetMapping
+    fun getAllIou(): PromissoryNotesDto {
```

Comment:
Более идиомично писать вот так:
```suggestion
    fun getAllIou(): PromissoryNotesDto = service.listPromissoryNotes()
```